docs/en/query_language/create.md

```diff
@@ -93,22 +93,23 @@ Besides default data compression, defined in [server settings](../operations/ser
 
 Supported compression algorithms:
 
-- `NONE` - no compression for data applied
+- `NONE` — no compression for data applied.
 - `LZ4`
-- `LZ4HC(level)` - (level) - LZ4\_HC compression algorithm with defined level.
+- `LZ4HC(level)` — (level) - LZ4\_HC compression algorithm with defined level.
```

Comment:
link to algorithm description

Reply:
In looks that there is no algorithm https://github.com/lz4/lz4/issues/660. Do we support it still?